### PR TITLE
FlagEval-debate blog

### PR DESCRIPTION
**Pull request by xuanricheng**
Congratulations! You've made it this far! Once merged, the article will appear at https://huggingface.co/blog. Official articles 
require additional reviews. Alternatively, you can write a community article following the process [here](https://huggingface.co/blog-explorers).

## Preparing the Article

You're not quite done yet, though. Please make sure to follow this process (as documented [here](https://github.com/huggingface/blog/tree/main?tab=readme-ov-file#how-to-write-an-article-)):

- [✅] Add an entry to [_blog.yml](https://github.com/huggingface/blog/blob/main/_blog.yml).
- [✅ ] Add a thumbnail. There are no requirements here, but there is a [template](https://github.com/huggingface/blog?tab=readme-ov-file#how-to-get-a-nice-responsive-thumbnail) if it's helpful.
- [✅] Check you use a short title and blog path.
- [✅ ] Upload any additional assets (such as images) to the Documentation Images [repo](https://huggingface.co/datasets/huggingface/documentation-images/tree/main/blog). This is to reduce bloat in the GitHub base repo when cloning and pulling. Try to have small images to avoid a slow or expensive user experience.
- [✅ ] Add metadata (such as authors) to your `md` file. You can also specify `guest` or `org` for the authors.
- [✅] Ensure the publication date is correct.
- [ ✅] Preview the content. A quick way is to paste the markdown content in https://huggingface.co/new-blog. **Do not click publish**, this is just a way to do an early check.

Here is an example of a complete PR: https://github.com/huggingface/blog/pull/2382

## Getting a Review

Please make sure to get a review from someone on your team or a co-author.
Once this is done and once all the steps above are completed, you should be able to merge.
There is no need for additional reviews if you and your co-authors are happy and meet all of the above.

Feel free to add @pcuenca as a reviewer if you want a final check. Keep in mind he'll be biased toward light reviews
(e.g., check for proper metadata) rather than content reviews unless explicitly asked.

### CODE REVIEW

**Comment by xuanricheng** [comment]
@xianbaoqian  Could you please review this PR? I forgot to add a reviewer earlier.

File: _blog.yml [hunk 1]
```diff
@@ -4976,4 +4976,14 @@
...
     - datasets
-    - guide
\ No newline at end of file
+    - guide
+
+- local: debate
+  title: "Letting Large Models Debate: The First Multilingual LLM Debate Competition"
+  author: xuanricheng
+  thumbnail: /blog/assets/debate/thumbnail.png
+  date: November 14, 2024
```
Comment (clefourrier) [inline _blog.yml:4985]: Data should be date of the release

File: _blog.yml [hunk 2]
```diff
@@ -4976,4 +4976,14 @@
...
     - research
     - datasets
-    - guide
\ No newline at end of file
+    - guide
+
+- local: debate
+  title: "Letting Large Models Debate: The First Multilingual LLM Debate Competition"
+  author: xuanricheng
+  thumbnail: /blog/assets/debate/thumbnail.png
```
Comment (clefourrier) [inline _blog.yml:4984]: Please use the "Leaderboards on the Hub" thumbnail (edited with your logo) and upload it in the correct folder

File: debate.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  org: BAAI
+- user: daiteng01
+  guest: true
+  org: BAAI
+- user: eyuansu71
+  guest: true
+  org: BAAI
+- user: Lyfly2024
+  guest: true
+  org: BAAI
```
Comment (clefourrier) [inline debate.md:33]: Tiezhen and I should be added as authors since we gave several reviews/rewrote sections.

File: debate.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+- **Lack of Discriminative Power**: Many model confrontations result in stalemates, lacking significant performance differentials. This necessitates a large number of user votes to ensure statistical stability and accuracy, thereby reducing evaluation efficiency and hindering rapid comparison of models’ relative strengths and weaknesses.
+- **Isolated Generation Phenomenon**: In these platforms, models do not truly interact; instead, they independently generate responses based on user inputs without engaging with each other’s outputs. This unidirectional generation relies on random user dialogues, making it difficult to probe the boundaries of models’ reasoning and logic, and challenging to evaluate their performance in complex adversarial contexts.
+- **Potential for Vote Bias**: Since models often exhibit distinctive styles, User votes usually favor specific model generation styles or formats rather than evaluating the content itself. Without expert annotations, these biases can skew the evaluation results, complicating an objective assessment of model capabilities and diminishing the reliability of user-driven judgments.
```
Comment (clefourrier) [inline debate.md:50]: User -> user

File: debate.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+  guest: true
+  org: BAAI
+---
+# Letting Large Models Debate: The First Multilingual LLM Debate Competition
+
+Current static evaluations and user-driven arenas have exhibited their limitations and biases in the previous year. Here, we explore a novel way to evaluate LLMs: debate.
+Debate is an excellent way to showcase reasoning strength and language abilities, used all across history, from the debates in the Athenian Ecclesia in the 5th century BCE to today's World Universities Debating Championship.
+Do today's large language models exhibit debate skills similar to humans? Which model is currently the best at debating? What can we learn from models when they debate against one another?
+To answer this question, BAAI has created a "Debate Arena", allowing large models to compete against each other. Currently, it supports debate competitions in English, Chinese, Arabic and Korean.
+<script type="module" src="https://gradio.s3-us-west-2.amazonaws.com/4.4.0/gradio.js"> </script>
```
Comment (clefourrier) [inline debate.md:41]: Are you using Gradio 4.4? You would want to upgrade if that's the case

File: debate.md [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+- **Potential for Vote Bias**: Since models often exhibit distinctive styles, User votes usually favor specific model generation styles or formats rather than evaluating the content itself. Without expert annotations, these biases can skew the evaluation results, complicating an objective assessment of model capabilities and diminishing the reliability of user-driven judgments.
+
+BAAI’s FlagEval Debate platform addresses these challenges by introducing genuine multi-model debates. Models engage in direct confrontations, showcasing their reasoning processes and depth. This allows evaluators to observe and compare differences in viewpoints, logical reasoning, and argumentation strategies among models. The platform’s multilingual support and real-time debugging capabilities enable users to study model strengths in realistic and interactive settings, ultimately providing more discriminative and in-depth evaluation results.
+## Key Features and Innovations of FlagEval Debate
```
Comment (clefourrier) [inline debate.md:53]: You need new lines before and after each title

File: debate.md [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+| 01.AI       | Yi-Lightning        | Self-debugged    |
+| Zhipu AI    | GLM-4-plus          | Provider-debugged|
+| DeepSeek    | DeepSeek_V2.5       | Self-debugged    |
+
+- **Self-debugged**: Denotes models we configured and optimized for debate.
+- **Provider-debugged**: Denotes models that were debugged and optimized by the model providers themselves.
+
+## Conclusion
+FlagEval Debate represents a significant advancement in LLM evaluation methodologies. By incorporating multilingual support, developer customization, and a dual evaluation system, it offers a robust framework for assessing models in interactive, real-world scenarios. Moving forward, BAAI is committed to refining this platform, and fostering innovation and standardization in AI evaluation practices. The goal is to provide developers and users with a forward-looking evaluation ecosystem, accelerating the evolution and deployment of advanced large language models.
+# About BAAI & FlagEval
```
Comment (clefourrier) [inline debate.md:124]: If you have sponsors/researchers who gave you a hand, you could add them to an Acknowledgment section

Comment (clefourrier) [reply debate.md:124]: Should be a title with ##

File: debate.md [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+The Beijing Academy of Artificial Intelligence (BAAI) was established in November 2018 as a new-type research institution, primarily focusing on original innovation and core technologies in artificial intelligence. Its aim is to drive revolutionary and disruptive breakthroughs in AI theory, methods, tools, systems, and applications.
+FlagEval, launched by BAAI in 2023, is a large model evaluation system that covers over 800 open-source and closed-source models globally. It includes more than 40 capability dimensions such as reasoning, mathematical skills, and task-solving abilities, along with five major tasks and four categories of metrics. In 2024, FlagEval successively launched the Colosseum and Debate Arena, platforms for model-to-model competition and battle.
+# References
```
Comment (clefourrier) [inline debate.md:127]: Should be a title with ##

File: debate.md [hunk 9]
```diff
@@ -0,0 +1,132 @@
...
+# About BAAI & FlagEval
+The Beijing Academy of Artificial Intelligence (BAAI) was established in November 2018 as a new-type research institution, primarily focusing on original innovation and core technologies in artificial intelligence. Its aim is to drive revolutionary and disruptive breakthroughs in AI theory, methods, tools, systems, and applications.
+FlagEval, launched by BAAI in 2023, is a large model evaluation system that covers over 800 open-source and closed-source models globally. It includes more than 40 capability dimensions such as reasoning, mathematical skills, and task-solving abilities, along with five major tasks and four categories of metrics. In 2024, FlagEval successively launched the Colosseum and Debate Arena, platforms for model-to-model competition and battle.
```
Comment (clefourrier) [inline debate.md:126]: Is FlagEval an organisation working on evaluation or an evaluation suite? If it's just an evaluation suite, I don't see the link with the blog and it should be removed. If it's an org/group focusing on evaluation, description should be adapted.

**Pull request closed without merging**